EventFiltering/PWGCF/CFFilterTwoN.cxx

```diff
@@ -0,0 +1,360 @@
+          registry.fill(HIST("fPtAntiProtonAfterSel"), antipd.pt());
+          Nantiproton++;
+        }
+      }
+      // select antideuterons
+      if (SelectParticlePID(antipd.pidcut(), std::vector<int>{o2::track::PID::Deuteron}, confNSigma, antipd.p(), confPIDThreshold)) {
+        registry.fill(HIST("fPtAntiDeuteronAfterSel"), antipd.pt());
+        Nantideuteron++;
+      }
+    }
+
+    if (KstarTrigger.value == 1 || KstarTrigger.value == 11) {
+      for (auto antilambda : partsAntiL) {
+        // select antilambdas
+        if (SelectParticlePID(antilambda.pidcut(), std::vector<int>{o2::track::PID::Lambda}, confNSigma, antilambda.p(), confPIDThreshold)) {
+          registry.fill(HIST("fPtAntiLambdaAfterSel"), antilambda.pt());
+          Nlambda++;
+        }
+      }
+    }
+
+    bool keepEvent[kLAST_CFTwoBodyTriggers] = {false, false};
+    int lowKstarPairs[kLAST_CFTwoBodyTriggers] = {0, 0};
+
+    // trigger for pd pairs
+    if (KstarTrigger.value == 0 || KstarTrigger.value == 11) {
+      if (Ndeuteron > 0 && Nproton > 0) {
+        for (auto& [p1, p2] : combinations(partsPD, partsPD)) {
+          if (!SelectParticlePID(p1.pidcut(), std::vector<int>{o2::track::PID::Proton, o2::track::PID::Deuteron}, confNSigma, p1.p(), confPIDThreshold)) {
+            continue;
+          }
+          if (closePairRejectionTT.isClosePair(p1, p2, partsFemto, magneticField)) {
+            continue;
+          }
+          auto kstar = FemtoDreamMath::getkstar(p1, mMassProton, p2, mMassDeuteron);
+          if (kstar < confKstarTriggerLimit.value) {
+            lowKstarPairs[kPD]++;
+            registry.fill(HIST("fKstarPD"), kstar);
+          }
+        }
+      }
+      if (Nantideuteron > 0 && Nantiproton > 0) {
+        for (auto& [p1, p2] : combinations(partsAntiPD, partsAntiPD)) {
+          if (!SelectParticlePID(p1.pidcut(), std::vector<int>{o2::track::PID::Proton, o2::track::PID::Deuteron}, confNSigma, p1.p(), confPIDThreshold)) {
+            continue;
+          }
+          if (closePairRejectionTT.isClosePair(p1, p2, partsFemto, magneticField)) {
+            continue;
+          }
+          auto kstar = FemtoDreamMath::getkstar(p1, mMassProton, p2, mMassDeuteron);
+          if (kstar < confKstarTriggerLimit.value) {
+            lowKstarPairs[kPD]++;
+            registry.fill(HIST("fKstarAntiPD"), kstar);
+          }
+        }
+      }
+    }
+
+    // trigger for ld pairs
+    if (KstarTrigger.value == 1 || KstarTrigger.value == 11) {
+      if (Ndeuteron > 0 && Nlambda > 0) {
+        for (auto& [p1, p2] : combinations(partsL, partsPD)) {
+          if (!SelectParticlePID(p1.pidcut(), std::vector<int>{o2::track::PID::Lambda, o2::track::PID::Deuteron}, confNSigma, p1.p(), confPIDThreshold)) {
+            continue;
+          }
+          if (closePairRejectionTT.isClosePair(p1, p2, partsFemto, magneticField)) {
```

Comment:
closePairRejectionTT is for track track, close pair rejection for v0-track must be initialised separately and used correctly

Reply:
fixed. I added the initalization inside the if statements of the global init function, so they QA object only get created if the specifig trigger is chosen.